sklearn/plot/_gridsearch_results.py

```diff
@@ -0,0 +1,143 @@
+import numpy as np
+from sklearn.plot import plot_heatmap
+
+
+def _plot_1D_results(cv_results, params, ax, xlabel, ylabel, title,
+                     fmt, xtickrotation):
+    import matplotlib.pyplot as plt
```

Comment:
I wouldn't go that far, but I'd include it in classes.rst. Also, the name doesn't start with an underscore, that means it's public.